modules/installation-creating-azure-dns.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-creating-azure-dns_{context}"]
+= Creating networking and load balancing components in Azure
+
+You must configure networking and load balancing in Microsoft Azure for your
+{product-title} cluster to use. One way to create these components is
+to modify the provided Azure Resource Manager (ARM) template.
+
+[NOTE]
+====
+If you do not use the provided ARM template to create your Azure infrastructure,
+you must review the provided information and manually create the infrastructure.
+If your cluster does not initialize correctly, you might have to contact Red Hat
+support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure an Azure account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VNet and associated subnets in Azure.
+
+.Procedure
+
+. Copy the template from the *ARM template for the network and load balancers*
+section of this topic and save it as `03_infra.json` on your computer. This
+template describes the networking and load balancing objects that your cluster
+requires.
+
+. Create the deployment by using the `az` CLI:
++
+----
+$ az group deployment create -g ${RESOURCE_GROUP} \
```

Comment:
It would be nice to be able to copy paste directly from that box and the other examples as well.

Current behavior for that one and many others:
`
>$ az group deployment create -g ${RESOURCE_GROUP} \
>   --template-file "03_infra.json" \
>   --parameters privateDNSZoneName="${CLUSTER_NAME}.${BASE_DOMAIN}" \ 
This command is implicitly deprecated because command group 'group deployment' is deprecated and will be removed in a future release. Use 'deployment group' instead.
Unable to parse parameter:`